README.md

```diff
@@ -1,54 +1,239 @@
-an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
-specific language governing permissions and limitations under the License.
+<br>
+
+###  <div align="left">Turns Data and AI algorithms into full web apps in no time. 
+###  How? Taipy GUI with Taipy Core pops out as a 360° platform to build production-ready web apps</div>  
+
+ 
+
+<br>
+<br>
 
-## Usage
-  - [Taipy](#taipy)
-  - [License](#license)
-  - [Usage](#usage)
-  - [What is Taipy](#what-is-taipy)
-  - [Installation](#installation)
-  - [Contributing](#contributing)
-  - [Code of conduct](#code-of-conduct)
-  - [Directory Structure](#directory-structure)
-## What is Taipy
+###  <div align="left">*Open Source, 100% Python*</div>
 
-Taipy is a Python library for creating Business Applications. More information on our [website](https://www.taipy.io).
-Taipy is split into multiple repositories to let users install the minimum they need.
 
-A more in depth documentation of taipy can be found [here](https://docs.taipy.io).
+<br>
+<br>
+<br>
+
+#  <div align="center"> 📊 We make both ends meet ⚙️ </div>  
+<br>
+ <div align="center">
+    
+| TAIPY GUI - the frond-end  | TAIPY Core - the back-end |
+| --------  | -------- |
+|<img src="readme_img/readme_gui_intro.gif" alt="Taipy Logo"  width="350" height="300"/> | <img src="readme_img/readme_core_intro.gif" alt="Taipy Logo"  width="350" height="300"/>
+
+    
+</div> 
+
+<br>
+<br>
 
 ## Installation
 
-Want to install _Taipy_? Check out our [`INSTALLATION.md`](INSTALLATION.md) file.
+Open a terminal and run:
+
+```bash
+$ pip install taipy
+```
+
+*You're all set! All aboard the Taipy journey 🚂*
+
+<br>
+
+## Ready, Set, GUI
+
+### Tiny Taipy GUI Demo
+
+```python
+excitement_page="""
+# Welcome to Taipy
+## Getting started with Taipy GUI
+### How excited are you to try Taipy?
+
+<|{excitement}|slider|min=1|max=100|>
+
+My excitement level: <|{excitement}|text|>
+"""
+excitement=100
+
+Gui(page=my_page).run()
+```
+*RUN*🏃🏽‍♀️  
+<div align="center">🎊 TADA! 🎊</div>  
+<br>
+<div align="center"><img src="readme_img/readme_gui_app.gif" width=600 height=400 alt="GUI demo"></img></div>  
+
+<br>
+<br>
+
+### <div align="center">*GUImme more*</div>
+*<div align="center">Check out our [Getting Started](https://docs.taipy.io/en/latest/getting_started/getting-started-gui/) and [Documentation](https://docs.taipy.io/en/latest/manuals/gui/)</div>*
+
+<br>
+<br>
+
+## EN-CORE?
+
+#### <div align="center">Let's create a back-end execution, also called scenario using Taipy Core. Our scenario will filter movie data based on the genre you choose. This scenario will be submitted (i.e., executed) each time the genre selection changes and output the seven most popular movies of that genre. </div>  
+<br>
+
+*Here is our filter function: it's a standard Python function that will constitute our unique task in the scenario*
```

Comment:
"it's a standard Python" -> "a standard Python"
"that will constitute our unique task in the scenario" -> "that is used by the unique task in the scenario"